bundles/org.openhab.binding.boschshc/README.md

```diff
@@ -228,6 +228,13 @@ This certificate is used for pairing between the Bridge and the Bosch Smart Home
 
 _Press and hold the Bosch Smart Home Controller Bridge button until the LED starts blinking after you save your settings for pairing_.
 
+### Supported Channels
+
+| Channel Type ID    | Item Type            | Writable | Description                                                        |
+|--------------------| -------------------- |:--------:|--------------------------------------------------------------------|
+| triggered-scenario | String               | &#9744;  | Name of the triggered scenario (e.g. by the Universal Switch Flex) |
```

Comment:
```suggestion
| triggered-scenario | String               | No | Name of the triggered scenario (e.g. by the Universal Switch Flex) |
```

Just keep it simple, no need for a checkbox of yes/no would do.

Reply:
Then we would have to change this in the whole readme consistently. Is there a documentation guideline saying how such yes/no columns should be rendered? I would propose to keep it for now and then change all values in a separate change to keep this pull request as small a possible.

Reply:
Yes consistency is important too. A follow up or would be great.